roles/as_code/lib/technical_principal.rb

```diff
@@ -0,0 +1,79 @@
+
+    ## What experience are we looking for?
+
+    We’ll expect you to be a polyglot programmer versed in a mix of object and functional programming languages. We’d expect you to have some blog posts about your discipline, perhaps even a talk or two. Not only this but the right person would be adept in sharing their knowledge with others – we’d love to hear some examples of coaching and growing team members.
+
+    We expect you to have experience architecting and leading software delivery teams, helping guide technical decision making in a servant leader approach. We expect you to have significant experience in designing and developing web applications in modern cloud-based environments.
+
+    Business savvy is a must. You will need experience in translating business goals and issues into technical solutions and have success stories to share. You will need to demonstrate experience working with senior IT stakeholders and working directly with CTO/COOs to create technical strategies.
+
+    ## What is it like to work at Made Tech?
+
+    We are a sanctuary for those wanting to hone their skills alongside like-minded learners. After joining our team it is common for new starters to comment on how much they’re learning and how much they enjoy the fact they are surrounded by people they can learn from. This includes our most senior hires.
+
+    The biggest thing you’ll take away from our culture after spending the day with us would be that we practice continuous improvement at every level. Everyone has peer-based one-to-ones every 2 weeks, along with monthly one-to-ones with a Line Manager. Teams have fortnightly retrospectives. We also hold a company wide retrospective fortnightly. We discuss our problems out in the open, and rather than punish failure we band together to find solutions.
+
+    Other notable things:
+
+    - Every Friday afternoon is dedicated to learning new skills
+    - Everyone is encouraged to write blog posts regularly
+    - Our handbook is open sourced
+    - We are vegan and non-drinker friendly as well as meat-eater and drinker friendly
+    - Retreats and trips every year
+
+    ## Salary
+
+    This role has a salary of £85-95k depending on experience.
+  DESCRIPTION
+
+  salary 85_000..95_000
+
+  expected 'to act as a role model and servant leader to their teams',
+    'We expect our Technical Principals to lead by example, to put the needs of others before their own, and to set and demonstrate a high standard for code and delivery quality. A Technical Principal is expected to show themselves to be highly reliable, to be on time, and to be well prepared.'
+
+  expected 'to coach and nurture Lead and Senior Engineers',
```

Comment:
Can see some overlap here with the more advanced learning/training/mentoring activities. 

Maybe there should be explicit mention about how we expect technical principals to work with #teampeople?

Reply:
We've removed code level mentoring.